src/components/caption/index.jsx

```diff
@@ -0,0 +1,27 @@
+import PropTypes from "prop-types";
+
+const COMPONENT_CLASS_NAME = "c-caption";
+
+const Caption = ({ caption, className, credit, title }) =>
+	!!(title || caption || credit) && (
+		<figcaption
+			className={className ? `${COMPONENT_CLASS_NAME} ${className}` : `${COMPONENT_CLASS_NAME}`}
+		>
+			{title && <span className="c-caption__title">{title} </span>}
```

Comment:
We want to make sure we are using ternary vs logical on rendering `{title ? <span></span> : null}`

Reply:
updated